packages/core/src/2d/assembler/SlicedSpriteAssembler.ts

```diff
@@ -118,7 +127,7 @@ export class SlicedSpriteAssembler {
     renderer._bounds.transform(worldMatrix);
   }
 
-  static updateUVs(renderer: SpriteRenderer): void {
+  static updateUVs(renderer: SpriteRenderer | UIImage): void {
     const subChunk = renderer._subChunk;
     const vertices = subChunk.chunk.vertices;
     const spriteUVs = renderer.sprite._getUVs();
```

Comment:
Above three lines can be optimized using destructuring to make it more concise and avoid repeated references to renderer and subChunk. Here's a suggested improvement:

const { _subChunk: { chunk: { vertices } }, sprite } = renderer;
const spriteUVs = sprite._getUVs();